packages/teleport/src/lib/tdp/client.ts

```diff
@@ -149,6 +154,24 @@ export default class Client extends EventEmitter {
     );
   }
 
+  handleMfaChallenge(buffer: ArrayBuffer) {
+    const mfaJson = this.codec.decodeMfaJson(buffer);
+    if (mfaJson.mfaType == 'n') {
+      this.emit(TermEventEnum.WEBAUTHN_CHALLENGE, mfaJson.jsonString);
+    } else {
+      // mfaJson.mfaType === 'u', or else decodeMfaJson would have thrown an error.
```

Comment:
Maybe you should explicitly check for u as it's one of the options in the type and then have a failure conditional for any other values.

Reply:
It's handled by the `try catch` now, as noted int he comment, there's no possibility of receiving anything else since the codec will throw an error if it's not 'n' or 'u'.